src/applications/vaos/components/ConfirmationRequestInfo.jsx

```diff
@@ -37,9 +37,10 @@ export default function ConfirmationRequestInfo({ data, facility }) {
         Your appointment request has been submitted
       </h1>
       <AlertBox status="success">
-        <strong>Your appointment request has been submitted.</strong> We're
-        reviewing your request. You don't have anything to do right now. A
-        scheduler will contact you to schedule the first available appointment.
+        <strong>Your appointment request has been submitted.</strong>
+        <br /> We're reviewing your request. You don't have anything to do right
```

Comment:
With the line break added, should there be a space before the `W`? Also I'm not sure if this should be enforced, but "We're" and "don't" could use the proper right single quote (`’`)

Reply:
Yep, I should add that.